lib/fastlane/plugin/wpmreleasetoolkit/helper/fastlane_metadata_file_writer.rb

```diff
@@ -0,0 +1,102 @@
+require 'fastlane_core/ui/ui'
+require 'fileutils'
+
+module Fastlane
+  module Helper
+    module FastlaneMetadataFilesWriter
+
+      # A model/struct defining a rule on how to process and map metadata from GlotPress into txt files
+      #
+      # @param [String] key The key in the GlotPress export for the metadata
+      # @param [Int] max_len The maximum length allowed by the App Store / Play Store for that key.
+      #        Note: If the translation for `key` exceeds the specified `max_len`, we will try to find an alternate key named `#{key}_short` by convention.
+      # @param [String] filename The (relative) path to the `.txt` file to write that metadata to
+      #
+      MetadataRule = Struct.new(:key, :max_len, :filename) do
```

Comment:
TODO: add `keyword_init: true` for the `Struct` so we can update all the `MetadataRule.new` call sites to use named parameters?